tests/data/test_parameter.py

```diff
@@ -0,0 +1,28 @@
+import pytest
```

Comment:
I would postpone to add this Parameter code until it will actually be used

Reply:
Ok